test/integration/example.test.js

```diff
@@ -209,6 +253,12 @@ describe('Creating a Docsify site (integration tests in Jest)', function () {
             example3 += 10;
             /// [something_else_not_code]
             console.log(example3);`,
+        '_media/example4.js': `
```

Comment:
Added this as a way check that e.g. `///` alone (without the `[demo]`) isn't producing anything